test/controllers/elastic_search_3_test.rb

```diff
@@ -116,7 +116,7 @@ def setup
     result = CheckSearch.new({ show: ['claims', 'links', 'images'] }.to_json)
     assert_equal 3, result.medias.count
     # filter by archived
-    pm.archived = true
+    pm.archived = 1
```

Comment:
Why not use the constants in the test too?

Reply:
Fixed by using constants